docs/UserGuide.md

```diff
@@ -116,7 +116,7 @@ Repo owners can provide the following additional information to RepoSense using
 * which git and github usernames belong to which authors
 * the display of an author
 
-To use this feature, add a `_reposense/config.json`  to the root of your repo using the format in the example below ([another example](../_reposense/config.json)):
+To use this feature, add a `_reposense/config.json`  to the root of your repo using the format in the example below ([another example](../_reposense/config.json)) and **commit it**:
```

Comment:
Should we mention this under `@@author` section too?

Reply:
Hmm... but its not really relevant because `@@author` can typically work with/without json file.

Reply:
But the tags will not affect the results unless they are committed first, right?

Reply:
oh... okay